src/main/java/org/sagebionetworks/bridge/sdk/BridgeUserClient.java

```diff
@@ -57,32 +55,22 @@
         super(session);
     }
 
-    /*
-     * UserProfile API
-     */
-    @Override
-    public UserProfile getProfile() {
+    public StudyParticipant getStudyParticipant() {
         session.checkSignedIn();
         
-        return get(config.getProfileApi(), UserProfile.class);
+        return get(config.getParticipantSelfApi(), StudyParticipant.class);
     }
-
-    @Override
-    public void saveProfile(UserProfile profile) {
+    
+    public void saveStudyParticipant(StudyParticipant participant) {
         session.checkSignedIn();
-        checkNotNull(profile, "Profile cannot be null.");
+        checkNotNull(participant, "StudyParticipant cannot be null.");
 
-        post(config.getProfileApi(), profile);
+        UserSession newSession = post(config.getParticipantSelfApi(), participant, UserSession.class);
+        session.setSharingScope(newSession.getSharingScope());
+        session.setDataGroups(newSession.getDataGroups());
+        session.setConsentStatuses(newSession.getConsentStatuses());
```

Comment:
Sync'ed up w/ Alx offline. While you can't consent through the Participants API, apparently by changing Data Group or Language, it's possible to suddenly become unconsented. Unfortunately, with our current test infrastructure, this edge case is difficult to test in our integration tests. I'm okay with not blocking on this.